kong/types.go

```diff
@@ -438,3 +438,18 @@ type DeveloperRole struct {
 	ID        *string `json:"id,omitempty" yaml:"id,omitempty"`
 	Name      *string `json:"name,omitempty" yaml:"name,omitempty"`
 }
+
+// Info represents the information concerning Kong.
+// +k8s:deepcopy-gen=true
+type Info struct {
+	Version       string                `json:"version,omitempty" yaml:"version,omitempty"`
+	Configuration *RuntimeConfiguration `json:"configuration,omitempty" yaml:"configuration,omitempty"`
+}
+
+// RuntimeConfiguration represents the runtime configuration of Kong.
+// +k8s:deepcopy-gen=true
+type RuntimeConfiguration struct {
+	Database string `json:"database,omitempty" yaml:"database,omitempty"`
+	Portal   bool   `json:"portal,omitempty" yaml:"portal,omitempty"`
+	RBAC     string `json:"rbac,omitempty" yaml:"rbac,omitempty"`
+}
```

Comment:
@mflendrich Should we consider simplifying this by moving the 3 fields in the RuntimeConfiguration struct to Info struct?
The extra struct seems unnecessary to me.

Reply:
Is there a way to do that without breaking JSON/YAML marshaling? If so, :100:. But, judging by the JSON marshaler spec, I don't know how to do that without an intermediate `struct`.

Reply:
This can probably be resolved async in a follow-up PR if necessary - that looks like a minor style issue. @hbagdi please speak up otherwise.

Reply:
Closing as (apparently) a minor style issue.